cgra/CgraRTL.py

```diff
@@ -94,9 +94,7 @@ def construct(s, CgraPayloadType,
     s.send_data_on_boundary_west  = [SendIfcRTL(DataType) for _ in range(height)]
 
     # Components
-    s.tile = [TileRTL(DataType, PredicateType, CtrlPktType,
-                      CgraPayloadType, CtrlSignalType,
-                      data_bitwidth,
+    s.tile = [TileRTL(CtrlPktType, CgraPayloadType,

```

Comment:
Can we pass in `IntraCgraPktType` here w/o `CgraPayloadType`? as `CgraPayloadType` is part of the `IntraCgraPktType`.